app/src/main/java/com/android/unio/model/firestore/FirestoreReferenceList.kt

```diff
@@ -75,7 +86,9 @@ class FirestoreReferenceList<T : UniquelyIdentifiable>(
    * @param uid The UID to remove.
    */
   override fun remove(uid: String) {
-    _uids.remove(uid)
+    if (_uids.remove(uid)) {
```

Comment:
I didn't know about List.remove returning a boolean that represents whether it made a change. Avoids an unnecessary operation :)

Reply:
Yeah that's crazy !